src/features/students/edit-students-details/edit-students-details.js

```diff
@@ -258,13 +258,13 @@ export const EditStudentsDetails = ({ id }) => {
                             {groups.map(({ id, name }) => (
                               <li
                                 className={classNames(styles['list-element'],
-                                  'd-flex bg-light border border-outline-secondary rounded')}
+                                  'd-flex bg-light border border-outline-secondary rounded font-weight-bold')}
```

Comment:
It looks better without `font-weight-bold`

Reply:
Thank's, done!